content/learning-paths/smartphones-and-mobile/using-neon-intrinsics-to-optimize-unity-on-android/9-summary.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Summary
+weight: 10
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+In this learning path you followed an iteration of measuring, optimizing and comparing changes using the Burst compiler and Neon intrinsics.
```

Comment:
This one also wants a title, but it could be something like "Adding Neon to Unity" ?